pandas/core/generic.py

```diff
@@ -2400,6 +2400,7 @@ def to_json(
         index: bool_t = True,
         indent: int | None = None,
         storage_options: StorageOptions = None,
+        mode: str = "w",
```

Comment:
Could you type this as `Literal["a", "w"]`?

Reply:
done.